indico/modules/events/registration/fields/base.py

```diff
@@ -102,6 +102,10 @@ def view_data(self):
     def get_friendly_data(self, registration_data):
         return registration_data.data
 
+    def get_places_used(self):
+        """Return number of places used if the field defines placesLimit"""
```

Comment:
"""Returns the number of used places for the field"""

We always return it, even if there's no place limit.